halfcar/road.py

```diff
@@ -40,8 +40,8 @@ class effectively slides a window of length `length` along that
         num_points = int(length * resolution)
 
         if x_min == None:
-            x_min = 0
-        x_max = x_min + length
+            x_min = -length / 2
```

Comment:
Not sure this is now correct. See PR #5

Reply:
Regarding `x_min` and `x_max` in `road.py`, my choice of `x_min = 0` as a default was fairly arbitrary, and I don't think the actual values matter (as long as `x_max` is equal to `x_min + length`) since the caller should be setting the appropriate values when instantiating the `Road` class.

Reply:
I need this for #5 too so close this PR

Note in Car you set x_min to a negative which is the reason I choose this.

```python
if road_func is None:
            # Initialize Road object. Because the car COG is centered at x = 0,
            # the road must extend left past the rear wheel point of contact
            # (x = -l_r) and right past the front wheel point of contact
            # (x = l_f). Choose road limits (road_x_min, road_x_max) accordingly.
            road_limits = (-2 * l_r, 2.5 * l_f)
```